pkgs/development/python-modules/awscrt/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildPythonPackage, fetchPypi, cmake, perl, gcc10, mock }:
```

Comment:
```suggestion
{ lib, buildPythonPackage, fetchPypi, cmake, perl, gcc, mock }:
```

Reply:
`gcc` didn't work locally for me, but will give this a go again